omod/src/main/webapp/resources/scripts/angular-common-error.js

```diff
@@ -0,0 +1,30 @@
+angular.module('uicommons.common.error', []). 
+	
+	factory('http-server-error-interceptor', function($q, $rootScope) {
+	    return {
+	        responseError: function(response) {
+            	$rootScope.response = response;
+	            if (response.status === 400) {
+	                $rootScope.$broadcast('event:server-validation-error');
+	            }
+	            else if (response.status === 500 || response.status === 405 || response.status === 410 // Gone
```

Comment:
For these other error codes, you need to show the value of the message field, if it is blank that is when you display the general message